interzeptor.go

```diff
@@ -0,0 +1,140 @@
+// Package interzeptors constructs a gRPC interceptor to enforce gNSI services.
+package interzeptors
+
+import (
+	"context"
+
+	apb "github.com/openconfig/gnsi/acctz"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/interzeptor/acctz"
+	"google.golang.org/grpc/interzeptor/authz"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	dlog = logrus.WithField("denvelope", "interzeptors")
+
+	zeptor *Interzeptor
```

Comment:
Can we document what this global is for?

Reply:
Actually I think interzeptor.go should not be part of this. I will remove them.

Reply:
Sorry -- bit confused -- where did the functionality in these files go? (Or are we wanting to review this elsewhere?)